src/applications/static-pages/events/components/Results/index.js

```diff
@@ -149,6 +149,18 @@ export const Results = ({
                     </p>
 
                     <div className="vads-u-display--flex vads-u-flex-direction--column">
+                      {event.fieldFacilityLocation?.entity?.entityUrl.path && (
+                        <p className="vads-u-margin--0">
+                          <a
+                            href={
+                              event.fieldFacilityLocation?.entity?.entityUrl
```

Comment:
Probably don't need the optional chaining here since it's inside a condition that already checked for it.  Probably an interesting debate about whether it's still better to have it, but I might fall on the side of not wanting the extra keystrokes, as a matter of habit.  I think it might help with clarity, as well, but that's arguable.